### ISSUE TITLE
Stop links in biographies opening in a new tab

### ISSUE THREAD

**Issue by pollecuttn** [wellcomecollection/wellcomecollection.org#7715]
**What**

Links in bios open in a new tab (unexpected behaviour); links in the main body of the text do not (expected behaviour).

**Where**

https://wellcomecollection.org/events/YhTyEhMAADOEddAT

https://wellcome.slack.com/archives/C8X9YKM5X/p1645780797455839 - there's already some investigation here.

**Comment by dannybirchall** [comment]
to add to our perplexities, exactly the same thing seems to be happening to the final two links on this page:

https://wellcomecollection.org/pages/YA64vRMAACAAgRjZ

**Comment by dannybirchall** [comment]
and also the links labelled 'conditions of use' and 'computer use policy' here 

https://preview.wellcomecollection.org/pages/Wuw19yIAAK1Z3Smm

**Comment by davidpmccormick** [comment]
Seems like Prismic are maybe trying to be helpful and adding this attribute to any links on an external domain. Can't find this documented anywhere though.

**Comment by davidpmccormick** [comment]
I've just deleted/re-added the problem link in the contributor bio on [this page](https://wellcomecollection.org/events/YhTyEhMAADOEddAT) and it is no longer opening in a new tab

**Comment by alexwlchan** [comment]
This might be something we could lint for if we ever do https://github.com/wellcomecollection/wellcomecollection.org/issues/7730
